internal/buildmanifest/manifest.go

```diff
@@ -0,0 +1,66 @@
+package buildmanifest
+
+import "gopkg.in/yaml.v2"
+
+const (
+	// Filename of a build manifest.
+	Filename = "build.yml"
+	// CurrentVersion os the manifest file format. See Manifest.SDK.
+	CurrentVersion = "2"
+)
+
+// Artifact is a file copied from one path to another during the build.
+type Artifact struct {
+	Path string `yaml:"path"`
+	Dest string `yaml:"dest"`
+}
+
+// Manifest used for the declarative build system for drivers.
+type Manifest struct {
+	// SDK version. Used to track future changes to this file. Current version is '2'.
```

Comment:
```suggestion
	// SDK version. Used to track future changes to this file. The current version is given by FileFormat.
```

But now I'm a little confused: Is this the SDK file format version or the SDK version? I think it's the first one (and hence my suggestion), but if that's correct the name `sdk` is a little odd. We might be stuck with the YAML name because of existing config files, but can we change the field name to reflect the semantics?

Reply:
It's an SDK version that first introduced the format. I changed the comment, hope it's better now.